app/views/PotentialUnderpaymentView.scala.html

```diff
@@ -20,62 +20,72 @@
 @import uk.gov.hmrc.play.views.html.layouts._
 @import uk.gov.hmrc.tai.config.ApplicationConfig
 @import includes.ptaHeader
+@import uk.gov.hmrc.tai.util.ViewModelHelper._
 @import uk.gov.hmrc.tai.util.{TaxYearRangeUtil => Dates}
 @import uk.gov.hmrc.tai.model.TaxYear
 @import uk.gov.hmrc.tai.viewModels.GoogleAnalyticsSettings
 @import uk.gov.hmrc.tai.util.TaxYearRangeUtil
 
-@this(mainTemplate: MainTemplate, loginStatus: LoginStatus)
+@this(main: MainTempleteNew, loginStatus: LoginStatus)
 
 @(viewModel:uk.gov.hmrc.tai.viewModels.PotentialUnderpaymentViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
         mainHeadingText = viewModel.pageTitle,
-        preHeadingText = Messages("tai.iya.tax.you.owe.preHeading")
-    )
+        preHeadingText = Messages("tai.iya.tax.you.owe.preHeading"),
+        displayBackLink = false,
+        showGdsVersion = true)
 }
 
-@mainTemplate(
-    title = viewModel.pageTitle,
-    articleClasses = Some("full-width"),
+@main(
+    title = messages(viewModel.pageTitle),
```

Comment:
why is this wrapped in messages()?

Reply:
I was following the pattern from my previous tickets. I can change this if this isn't standard.

Reply:
does it work? I would of thought this looks for a message key?

Reply:
It works, no tests surrounding the title seems to fail, and this is used in the DOM correctly too.